generator.go

```diff
@@ -405,14 +406,19 @@ func (g *Generator) generateModelFile() error {
 	if err := os.MkdirAll(modelOutPath, os.ModePerm); err != nil {
 		return fmt.Errorf("create model pkg path(%s) fail: %s", modelOutPath, err)
 	}
+	p, err := packages.Load(&packages.Config{Dir: modelOutPath})
```

Comment:
`packages.Load` cannot get package path if there is no file in directory(just make that dir)